packages/service-integration/src/service_integration/compose_spec_model.py

```diff
@@ -0,0 +1,559 @@
+# generated by datamodel-codegen:
+#   filename:  compose-spec.json
+#   timestamp: 2021-11-12T17:03:07+00:00
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+
+from pydantic import BaseModel, Extra, Field, conint, constr
+
+# port number range
+# "format": "ports"
+PortInt = conint(gt=0, lt=65535)
+# TODO: format "duration" https://schema.org/Duration
+# TODO: format "expose"
+
+
+class Config1(BaseModel):
```

Comment:
this is auto-generated? what is Config1 for ?
absolutely no documentation... it's a bit rough I'd say

Reply:
I separated the auto-generated module and created a facade that makes more clear the design
in addition, there one can add aliases